ui/app/AppLayouts/Browser/stores/Web3ProviderStore.qml

```diff
@@ -18,6 +21,10 @@ QtObject {
         dappPermissionsModule.addPermission(permission, hostName)
     }
 
+    function hasWalletConnected(hostname) {
+        return dappPermissionsModule.hasPermission(hostname, "web3")
+    }
+
```

Comment:
@alaibe would this be the right way to do it?

Reply:
yes